From ddd9bbcfd5081573a82d203e0e008bb970393207 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 15 Mar 2019 22:38:10 -0700
Subject: [PATCH] TRITON-884 remove cloud analytics (ca) from Triton (missing
 CA ref in test setup) Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
 Approved by: Marsell Kukuljevic <marsell@joyent.com>

---
 test/runtests | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/test/runtests b/test/runtests
index f0cb2fa..b8e13e3 100755
--- a/test/runtests
+++ b/test/runtests
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -115,7 +115,6 @@ fi
 if [[ -n "$opt_headnode" ]]; then
     export AMON_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json amon_admin_ips | cut -d, -f1')
     export CNAPI_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json cnapi_admin_ips | cut -d, -f1')
-    export CA_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json ca_admin_ips | cut -d, -f1')
     export IMGAPI_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json imgapi_admin_ips | cut -d, -f1')
     export NAPI_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json napi_admin_ips | cut -d, -f1')
     export VMAPI_IP=$(ssh $opt_headnode 'bash /lib/sdc/config.sh -json | json vmapi_admin_ips | cut -d, -f1')
@@ -130,7 +129,6 @@ if [[ -n "$opt_headnode" ]]; then
 else
     export AMON_IP=$(bash /lib/sdc/config.sh -json | json amon_admin_ips | cut -d, -f1)
     export CNAPI_IP=$(bash /lib/sdc/config.sh -json | json cnapi_admin_ips | cut -d, -f1)
-    export CA_IP=$(bash /lib/sdc/config.sh -json | json ca_admin_ips | cut -d, -f1)
     export NAPI_IP=$(bash /lib/sdc/config.sh -json | json napi_admin_ips | cut -d, -f1)
     export VMAPI_IP=$(bash /lib/sdc/config.sh -json | json vmapi_admin_ips | cut -d, -f1)
     export UFDS_ADMIN_UUID=$(bash /lib/sdc/config.sh -json | json ufds_admin_uuid)
@@ -147,7 +145,6 @@ fi
 echo ""
 echo "# Datacenter config:"
 echo "#     AMON_IP is $AMON_IP"
-echo "#     CA_IP is $CA_IP"
 echo "#     CNAPI_IP is $CNAPI_IP"
 echo "#     CNS_IP is $CNS_IP"
 echo "#     IMGAPI_IP is $IMGAPI_IP"
-- 
2.21.0

